api/src/test/scala/hmda/api/http/InstitutionsHttpApiSpec.scala

```diff
@@ -1,196 +1,20 @@
 package hmda.api.http
 
-import java.io.File
-
-import akka.actor.ActorRef
 import akka.event.{ LoggingAdapter, NoLogging }
-import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
 import akka.http.scaladsl.model._
 import akka.http.scaladsl.model.headers.RawHeader
 import akka.http.scaladsl.server.{ MethodRejection, Route }
-import akka.http.scaladsl.testkit.ScalatestRouteTest
-import akka.pattern.ask
 import akka.util.Timeout
-import com.typesafe.config.ConfigFactory
-import hmda.api.RequestHeaderUtils
-import hmda.api.model._
-import hmda.model.fi._
-import hmda.model.institution.Institution
-import hmda.persistence.HmdaSupervisor
-import hmda.persistence.HmdaSupervisor.{ FindActorByName, FindSubmissions }
-import hmda.persistence.demo.DemoData
-import hmda.persistence.institutions.SubmissionPersistence.UpdateSubmissionStatus
-import hmda.persistence.institutions.{ InstitutionPersistence, SubmissionPersistence }
-import org.iq80.leveldb.util.FileUtils
-import org.scalatest.{ BeforeAndAfterAll, MustMatchers, WordSpec }
 
 import scala.concurrent.duration._
 
-class InstitutionsHttpApiSpec extends WordSpec with MustMatchers with ScalatestRouteTest
-    with InstitutionsHttpApi with BeforeAndAfterAll with RequestHeaderUtils {
+class InstitutionsHttpApiSpec extends InstitutionSpec {
```

Comment:
It looks like all that's left in this file is authorization-related specs. If that's true, could we rename the spec to be more specific, like `InstitutionsAuthSpec` or similar?